### ISSUE TITLE
SLE-Micro

### ISSUE THREAD

**Issue by rjschwei** [SUSE-Enceladus/keg-recipes#245]
From the latest expansion for 5.5 image builds:

Typo: sle-micro-sysconig

We do no longer need/want WICKED_DEBUG SLE-Micro should be completely on NetworkManager?

We can skip the NCURSES setting, no YaST in SLE-Micro

We can skip the locks, no plymouth-* in SLE-Micro

**Comment by jgleissner** [comment]
I don't think we plymouth in any image. Should we just drop the zypp lock entirely?

**Comment by rjschwei** [comment]
No, we cannot drop the lock anywhere else because plymouth is in the repos and a user may pull it inadvertently basically breaking their instance. That said having the lock doesn't hurt, maybe I am being too picky.

**Comment by jgleissner** [comment]
Wrt NCURSES setting, this isn't really specific to YaST. Apparently putty (which is a popular Windows SSH client) had issues drawing lines in UTF8 mode (which YaST uses) but support for this was implemented 5 years ago according to https://git.tartarus.org/?p=simon/putty.git;a=commitdiff;h=6afa955a2ebf017755af134264a9423f54b9351c so perhaps we can just drop this altogether.

On a side note, we install `yast2-schema-micro` in SLE Micro images (instead of `yast2-schema-default` as in everything else). I was wondering why there even was a `yast2-schema-micro` but it appears the yast2 schema packages are really just for AutoYaST so I suppose their inclusion is pointless and we can just drop them.
